### PR TITLE
Sample code

### PR DESCRIPTION
**Pull request by ruchabhange**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,4 @@
-# Terraform-HCL-generation-tools
\ No newline at end of file
+# Terraform-HCL-generation-tools
+
+This repository hosts sample Terraform HCL code generated with different tools from existing cloud resources. Primary purpose of creating the same is to host the code here for reference in blog.
+
```
Comment (techmaharaj) [inline README.md:4]: Can we add List of tools along with links to the respective folder? Something like below:

- Aztfy
- Terracognita
- Terraform-Import
- Terraformer

And once the blog post is live, please add the link to the blog post in this readme.

File: aztfy/README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Aztfy: POC
+
+This folder contains sample Terraform configuration files generated with Aztfy for Azure cloud virtual machine.
+
+
```
Comment (techmaharaj) [inline aztfy/README.md:5]: Let us add a link to the tool's page/documentation in the readme. That way if anyone wants to know more about it, they can simply visit the link.

File: terraform-import/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Terraform Import: POC
+
+This folder contains sample Terraform state file generated with Terraform Import for AWS VPC.
+
+
```
Comment (techmaharaj) [inline terraform-import/README.md:5]: Let us add a link to the tool's page/documentation in the readme. That way if anyone wants to know more about it, they can simply visit the link.

File: terraformer/README.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Terraformer: POC
+
+This folder contains sample Terraform file generated with Terraformer for AWS VPC.
+
+
```
Comment (techmaharaj) [inline terraformer/README.md:5]: Let us add a link to the tool's page/documentation in the readme. That way if anyone wants to know more about it, they can simply visit the link.

**Comment by nimblenitin** [comment]
merging after including comment req.

**Pull request merged**